### PR TITLE
Fixes resize image based on placeholder

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
Fixes resize image based on placeholder

Changes:

Reduced image sizes on server.

Media:

Screen Shot 2023-02-01 at 1 32 40 PM

fixes https://github.com/EveripediaNetwork/issues/issues/998
fixes https://github.com/EveripediaNetwork/issues/issues/1002

### CODE REVIEW

File: src/components/Wiki/WikiPreviewCard/WikiPreviewCard.tsx [hunk 1]
```diff
@@ -52,7 +52,13 @@ const WikiPreviewCard = ({
       cursor="pointer"
     >
       <AspectRatio w="100%" ratio={WIKI_IMAGE_ASPECT_RATIO}>
-        <WikiImage imageURL={getWikiImageUrl(wiki.images)} alt={wiki.title} />
+        <Image
+          src={getWikiImageUrl(wiki.images)}
+          alt={wiki.title}
+          boxSize="100%"
+          imgH={IMAGE_BOX_SIZE}
+          imgW={IMAGE_BOX_SIZE}
```
Comment (Royal-lobster) [inline src/components/Wiki/WikiPreviewCard/WikiPreviewCard.tsx:60]: @Adebesin-Cell may be we can create a new prop something like imgBoxSize on Image component.

**Comment by Adebesin-Cell** [comment]
@Softdev1 @Royal-lobster I think this PR should fix Aspect ratios too.

File: src/components/Elements/Image/Image.tsx [hunk 2]
```diff
@@ -4,28 +4,26 @@ import NextImage, { ImageProps } from 'next/image'
...
   blurDataURL,
   hideOnError,
   sizes,
   objectFit = 'cover',
+  imgBoxSize,
   ...rest
 }: NextChakraImageProps) => (
   <Box
-    h={`${imgH}px`}
-    w={`${imgW}px`}
```
Comment (Royal-lobster) [inline src/components/Elements/Image/Image.tsx:28]: Oh no, these should be same. we should allow devs to add imgH and imgW seperately. also imgBoxSize if both are same. 

you can do something like 

```
h={`${imgBoxSize || imgH}`}
w={`${imgBoxSize || imgW}`}
```

File: src/components/Elements/Image/Image.tsx [hunk 3]
```diff
@@ -51,8 +49,8 @@ export const Image = ({
       placeholder={placeholder}
       blurDataURL={blurDataURL}
       loading={priority ? 'eager' : 'lazy'}
-      width={imgW}
-      height={imgH}
```
Comment (Royal-lobster) [inline src/components/Elements/Image/Image.tsx:55]: Oh no, these should be same. we should allow devs to add imgH and imgW seperately. also imgBoxSize if both are same.

you can do something like

height={`${imgBoxSize || imgH}`}
width={`${imgBoxSize || imgW}`}

Comment (Adebesin-Cell) [reply src/components/Elements/Image/Image.tsx:55]: Oh! I'll add the fix for that now.

File: src/components/About/BlogCard.tsx [hunk 4]
```diff
@@ -25,7 +26,13 @@ const BlogCard = ({
         overflow="hidden"
       >
         <Box h="210px" mt={-6} mx={-6} mb={6} pos="relative">
-          <Image src={image} alt={title} layout="fill" />
+          <Image
+            w="full"
+            src={image}
+            alt={title}
+            layout="fill"
+            imgBoxSize={IMAGE_BOX_SIZE}
```
Comment (Royal-lobster) [inline src/components/About/BlogCard.tsx:34]: @Adebesin-Cell we need to give exact image dimensions so nextjs know how much space it takes on page even though it renders smaller image. the blog card is more of a rectangle. inspect the size from the dev tools and fill in imgH and imgW

**Review by Softdev1 — APPROVED** [review]

**Comment by Royal-lobster** [comment]
Also this needs to be addressed https://github.com/EveripediaNetwork/ep-ui/pull/1106#discussion_r1093177681

File: src/components/About/BlogCard.tsx (hunk 4 above)
Comment (Adebesin-Cell) [reply src/components/About/BlogCard.tsx]: > @Adebesin-Cell we need to give exact image dimensions so nextjs know how much space it takes on page even though it renders smaller image. the blog card is more of a rectangle. inspect the size from the dev tools and fill in imgH and imgW

For this I tried making a rectangle by using the imgW= {IMAGE_BOX_SIZE * WIKI_IMAGE_ASPECT_RATIO} and imgH={IMAGE_BOX_SIZE}.

**Review by Royal-lobster — APPROVED** [review]

**Pull request merged**